sdg_mapping/sdg_index/sdg_index_utils.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
+import pandas as pd
+
+import sys
+import os
+import json
+
+from sdg_mapping import project_dir
+
+with open(f'{project_dir}/data/aux/sdg_index_data_opts.json', 'r') as f:
+    state_dict = json.load(f)
+
+
+def sdg_index_file_path(year):
+    """sdg_index_file_path
+    To fetch filepath of SDG Index datasets.
+
+    Args:
+        year (str): Year of SDG Index datasets
+
+    Returns:
+        (str): File path
+
+    """
+    fname = f'{year}_sdg_index.xlsx'
+    return f'{project_dir}/data/raw/{fname}'
+
+
+def read_workbook(data_path, state):
+    """read_workbook
+    To read an Excel read_workbook

```

Comment:
Can this describe the function a bit more? `pd.read_excel` reads an Excel workbook. What does this function do on top of that which is specific to the use of this module?